recipes/deepdiff/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "deepdiff" %}
+{% set version = "3.3.0" %}
+{% set md5 = "d16022fbf7b6a1ad401c629b81088614" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  noarch: python
```

Comment:
If `idna` is in fact a dependency (see other comment) that is platform specific, then `noarch` isn't appropriate:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages